### PR TITLE
[WIP] Very WIP - Transfer detail layouts

### PR DESCRIPTION
**Pull request by marcelinkaaa**
Issue #581. This is in a very beginning stages. 
To talk over - the overall set up, having an activity that opens a fragment (I see you guys have been doing that before so just wanted to follow this route), the layouts still need some work - too much nesting on one of the fragments (which of course we can easily fix by adding new info programmatically) etc., some cleaning up needed at later date.  
Also nothing is connected yet to the actual data as I still did not have a chance to talk to anyone with the info we really want to include and info we don't need to have or would be too much to include now. The design is just based on my research in terms of what other companies already show their users. Of course we can add/remove some of them. 

Basically wanted to talk it over first :) so I know what is the final path.

### CODE REVIEW

File: android/src/com/frostwire/android/gui/activities/TransferDetailActivity.java [hunk 1]
```diff
@@ -40,39 +37,16 @@ public TransferDetailActivity() {
...
         mViewPager.setAdapter(mSectionsPagerAdapter);
 
-        TabLayout tabLayout = (TabLayout) findViewById(R.id.activity_transfer_detail_tab_layout);
+        TabLayout tabLayout = findViewById(R.id.transfer_detail_tab_layout);
         tabLayout.setupWithViewPager(mViewPager);
     }
 
-    @Override
-    public boolean onPrepareOptionsMenu(final Menu menu) {
-        getMenuInflater().inflate(R.menu.fragment_transfer_detail_menu, menu);
```
Comment (gubatron) [inline android/src/com/frostwire/android/gui/activities/TransferDetailActivity.java]: where did the menu go?

File: android/src/com/frostwire/android/gui/activities/TransferDetailActivity.java [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        mViewPager = (ViewPager) findViewById(R.id.activity_transfer_detail_viewpager);
+        mViewPager.setAdapter(mSectionsPagerAdapter);
+
+        TabLayout tabLayout = (TabLayout) findViewById(R.id.activity_transfer_detail_tab_layout);
+        tabLayout.setupWithViewPager(mViewPager);
+    }
+
+    @Override
+    public boolean onPrepareOptionsMenu(final Menu menu) {
+        getMenuInflater().inflate(R.menu.fragment_transfer_detail_menu, menu);
```
Comment (gubatron) [inline android/src/com/frostwire/android/gui/activities/TransferDetailActivity.java]: i suppose it went here (the menu)

File: android/res/layout/fragment_transfer_detail_details.xml [hunk 3]
```diff
@@ -0,0 +1,794 @@
...
+
+            <TextView
+                style="@style/transferDetailText"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/transfer_detail_comment_dummy" />
+        </LinearLayout>
+    </LinearLayout>
+</ScrollView>
+
```
Comment (gubatron) [inline android/res/layout/fragment_transfer_detail_details.xml]: that's one long layout... hmm

**Comment by gubatron** [comment]
This looks good, lot's of work, way to go.

However, codacy/pr shows a few warnings you should take a look at
https://www.codacy.com/app/contact_68/frostwire/pullRequest?prid=937240

**Comment by aldenml** [comment]
as a general comment, inherit from `AbstractFragment` (instead of `Fragment`) and use `findView` (instead of `findViewById`)

File: android/res/layout/fragment_transfer_detail_details.xml (hunk 3 above)
Comment (marcelinkaaa) [reply android/res/layout/fragment_transfer_detail_details.xml]: As per my comment above "too much nesting on one of the fragments" - so I was aware and I would like to later populate many of those programatically. This was just in the beginning to show you how the info would look. Lets go over all the torrent info you would like to include and finalize the design first - I will start on making all the necessary changes right after.

**Comment by marcelinkaaa** [comment]
@aldenml yes, thanks will correct tomorrow.

File: android/src/com/frostwire/android/gui/activities/TransferDetailActivity.java [hunk 4]
```diff
@@ -91,15 +92,15 @@ public SectionsPagerAdapter(FragmentManager fm) {
         }
 
         @Override
-        public Fragment getItem(int position) {
-            Fragment fragment;
+        public AbstractFragment getItem(int position) {
```
Comment (gubatron) [inline android/src/com/frostwire/android/gui/activities/TransferDetailActivity.java]: 👍

**Comment by gubatron** [comment]
opening a new pull request with another branch
